extensions/wikia/PhalanxII/PhalanxStatsSpecialController.class.php

```diff
@@ -26,34 +26,34 @@ public function index() {
 		if ( strpos( $par, 'wiki' ) === 0 ) {
 			// show per-wiki stats
 			list ( , $wikiId ) = explode( "/", $par, 2 );
-			$this->blockWikia($wikiId);
+			$this->blockWikia( $wikiId );
 		} else {
-			$blockId = $this->wg->Request->getInt('blockId', intval($par));
+			$blockId = $this->wg->Request->getInt( 'blockId', intval( $par ) );
 			if ( !empty( $blockId ) ) {
 				// show block stats
-				$this->blockStats($blockId);
+				$this->blockStats( $blockId );
 			} else {
 				// show help page
-				$this->forward('PhalanxStatsSpecial', 'help');
+				$this->forward( 'PhalanxStatsSpecial', 'help' );
 			}
 		}
 
 		wfProfileOut( __METHOD__ );
 	}
 
-	private function blockStats($blockId) {
+	private function blockStats( $blockId ) {
 		$out = $this->getOutput();
 		$out->setPageTitle( $this->msg( 'phalanx-stats-title' )->text() . "#$blockId" );
 		$out->addBacklinkSubtitle( $this->title );
 
-		$data = Phalanx::newFromId($blockId);
+		$data = Phalanx::newFromId( $blockId );
 
 		if ( !isset( $data["id"] ) ) {
 			$out->addWikiMsg( 'phalanx-stats-block-notfound', $blockId );
 			return false; // skip rendering
 		}
 
-		$data['author_id'] = User::newFromId($data['author_id'])->getName();
+		$data['author_id'] = User::newFromId( $data['author_id'] )->getName();
 		$data['timestamp'] = $this->getLanguage()->timeanddate( $data['timestamp'] );
 
 		if ( $data['expire'] == null ) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "expire" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)